helm_deploy/dps-smoketest/values.yaml

```diff
@@ -42,4 +42,4 @@ generic-service:
 generic-prometheus-alerts:
   targetApplication: dps-smoketest
   alertSeverity: syscon
-  sqsAlertsTotalMessagesThreshold: 1
+  sqsAlertsTotalMessagesThreshold: 50
```

Comment:
alert went off this am.  Default is 100 but I changed it to 1 on last PR.  Bumping back up to prevent false positives.